tests/Feature/Api/ApiTestCase.php

```diff
@@ -2,14 +2,39 @@
 
 namespace Tests\Feature\Api;
 
+use Illuminate\Foundation\Testing\TestResponse;
 use Tests\TestCase;
+use Hedonist\Entities\User\User;
+use Illuminate\Contracts\Auth\Authenticatable as UserContract;
 
 /**
  * Class ApiTestCase
  * @package Tests\Feature\Api
  */
 abstract class ApiTestCase extends TestCase
 {
+    protected $authenticatedUser;
+
+    public function actingAs(UserContract $user, $driver = null) : self
```

Comment:
Why public method?

Reply:
Laravel native trait this method is `public` for some reasons

`trait InteractsWithAuthentication`
`{`
`    /**`
`     * Set the currently logged in user for the application.`
`     * @param  \Illuminate\Contracts\Auth\Authenticatable  $user`
`     */`
`    public function actingAs(UserContract $user, $driver = null)`
`    {`
`        $this->be($user, $driver);`
`        return $this;`
`    }`